### DISCUSSION TITLE
Unexpected taskbar icon hovering highlight

### DISCUSSION THREAD

**Discussion by Alan-Kuan** [home-sweet-gnome/dash-to-panel#2065] [Q&A] [answered]
After I updated the extension to the latest version that supports Gnome 46, when I hover my cursor on any icon on my taskbar, a rounded light gray highlight is displayed. It didn't happen before the update.
Besides, I cannot find where I can turn it off or configure this.

<img src="https://github.com/home-sweet-gnome/dash-to-panel/assets/24734750/63ce202c-12e8-49ec-8534-974e02c416d4" height="500" />

Is there anyone encounter this problem? How can I solve it?

**Comment by charlesg99** [comment, marked as answer]
Yeah this is due to some css changes in gnome-shell 46. This is fixed on the master branch and will be released soon, just waiting a few days in case anything else pops up.

**Reply by Alan-Kuan** [reply]
I see. Thanks for your reply.
